### PR TITLE
My protfolio site(mobile-first) final code, HTML,CSS and Readme file …

### PR DESCRIPTION
**Pull request by emhamza**
In this part of the project, I have modified and updated different the HTML, CSS and Readme files according to my Figma template. I have managed to use different properties and tools of CSS e.g. flexbox property, grid property, and HTML5 tools to picture my portfolio professionally.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -37,6 +37,128 @@ <h1 class="item2">I'm a software developer</h1>
...
+         <div class="a1">JavScript</div>
+         <div class="a1">html</div>          
+     </div>
+      <div class="b_div"><button class="button">See Project</button></div>
+    </div>
+
+   </section>
+
+    <section class="about_me_container">
+      <h1 class="about_me1">About me</h1>
```
Comment (NahnahAJ) [inline portfolio.html:154]: - [ ] Lastly, the spacing between the `works-section `and the `about-me section `isn't similar to the Figma design. It is smaller compared to the Figma design.  Please try to stick to the design guidelines as much as possible so you can fix this as well. You can increase the `margin-top `of the` about-me `section to fix this :wink: 
- Your Design
image

- The Figma Design
image

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED
Hi @emhamza , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] No linter errors :heavy_check_mark: 
- [ ] Good use of GitHub flow :heavy_check_mark: 
- [ ] Nice Design :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NahnahAJ** [comment]
- [ ] You're also missing the last cards of the `about-section`. Kindly add these as well. I have attached a video below fo your reference. 
- Kindly add the Languages Skills and Frameworks cards :+1: 
- Video Reference
[Screencast from 02-18-2023 09:22:47 AM.webm](https://user-images.githubusercontent.com/102699958/219852698-06d5d7d3-6862-4ee4-a370-33901f86e860.webm)

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @emhamza , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image


## Highlights 
:+1: no linter errors
:+1: the design looks good
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emhamza** [comment]
My portfolio website (mobile-first) is ready and approved by the code reviewer. I made some changes to it since the last code review which includes changing of button size and adding new cards in the about me section along with other miner changes.

**Pull request merged**